LibraBFT/Impl/Consensus/SafetyRules/Properties/SafetyRules.agda

```diff
@@ -269,22 +272,26 @@ module ConstructAndSignVoteM where
 
      ∀ r → r ≡ pre →
        RWST-weakestPre
-         (RWST-put "lSafetyData ∙= (safetyData1 & sdLastVote ?~ vote)" pre)
+         (RWST-put (r & lSafetyData ∙~ ("safetyData1" & sdLastVote ?~ "vote")))
```

Comment:
With the new shorthand, the substitution of `r` (which is a fairly important point, I think) into the function body becomes clear.